libcloud/dns/drivers/cloudflare.py

```diff
@@ -317,14 +317,20 @@ def create_record(self, name, zone, type, data, extra=None):
         or iodef.
 
         For example: 0 issue test.caa.com
+
+        NOTE: For SSHFP RecordType, data need to be in the format:
+        <algorithm> <type> <fingerprint>
         """
         url = '{}/zones/{}/dns_records'.format(API_BASE, zone.id)
 
-        data = self._normalize_record_data_for_api(type=type, data=data,)
+        content, data = self._normalize_record_data_for_api(
+            type=type, data=data
+        )
         body = {
             'type': type,
             'name': name,
-            'content': data,
+            'content': content,
+            'data': data
```

Comment:
I assume this will work fine and API won't return an error if ``data`` is empty?

Or perhaps, to be on the safe side, we should just exclude ``data`` attribute if it's an empty dict?